config/src/main/java/com/typesafe/config/impl/ConfigNumber.java

```diff
@@ -93,6 +94,11 @@ static ConfigNumber newNumber(ConfigOrigin origin, long number,
             return new ConfigLong(origin, number, originalText);
     }
 
+    static ConfigNumber newNumber(ConfigOrigin origin, BigInteger number,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/hp/config/file/2444185970/issues/source?bid=3245965&fileBranchId=3245975#l97)